nodes/object_operators/ScGetChildren.py

```diff
@@ -0,0 +1,27 @@
+import bpy
+
+from bpy.types import Node
+from .._base.node_base import ScNode
+from ...helper import focus_on_object
+
+class ScGetChildren(Node, ScNode):
+    bl_idname = "ScGetChildren"
+    bl_label = "Get Children"
+    
+    def init(self, context):
+        super().init(context)
+        self.inputs.new("ScNodeSocketObject", "Object")
+        self.outputs.new("ScNodeSocketArray", "Children")
+
+    def error_condition(self):
+        return (
+            self.inputs["Object"].default_value == None
+        )
+
+    def pre_execute(self):
+        focus_on_object(self.inputs["Object"].default_value)
```

Comment:
This is not important if you just want to output a list

Reply:
Fixed.